shimmingtoolbox/unwrap/unwrap_phase.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+""" Wrapper to different unwrapping algorithms
+
+"""
+
+
+import numpy as np
+from shimmingtoolbox.unwrap.prelude import prelude
+
+
+def unwrap_phase(complex_array, affine, unwrap_function="prelude", mask=np.array([-1])):
+    """ Calls different unwrapping algorithms according to the specified `unwrap_function` parameter
+
+    Args:
+        complex_array (numpy.ndarray): 2D,3D,4D or 5D complex values numpy array to perform phase unwrapping
+        affine (numpy.ndarray): 2D array (4x4) containing the transformation coefficients. Can be acquired by :
```

Comment:
looks strange to ask for affine transfo (supposed to be for prelude, but unclear here)

Reply:
unwrap_phase function removed for now